### PR TITLE
AC Test Fix - LRAC-9714 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-9714
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1578469397

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by AfonsoHSantos — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase [hunk 1]
```diff
@@ -58,7 +58,7 @@ definition {
 		task ("Change the timezone to Log Angeles of United States") {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase:58]: Hey @ViniLopes20, this `task ("Change the timezone to Log Angeles of United States")` I think it's `Los` Angeles

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**